src/server/src/modules/document.ts

```diff
@@ -0,0 +1,92 @@
+import { Document } from '@entities/document';
+import { Student } from '@entities/student';
+import { getRepository } from 'typeorm';
```

Comment:
If after addressing all the changes requested by the comments, this getRepository import is no longer needed, please remove it.